Sources/App/Models/Version.swift

```diff
@@ -90,6 +90,9 @@ final class Version: Model, Content {
     @Field(key: "url")
     var url: String?
 
+    @Field(key: "has_binary_targets")
+    var hasBinaryTargets: Bool
+    
```

Comment:
Nit: we're not consistent with it everywhere but we try to keep properties within their "groups" (data fields, relationships) sorted alphabetically. I believe `Version` is one of the types where it's sorted.